templates/field/field--field-secure-attachment.html.twig

```diff
@@ -0,0 +1,66 @@
+{#
+/**
+ * @file
+ * Theme override for a field.
+ *
+ * To override output, copy the "field.html.twig" from the templates directory
+ * to your theme's directory and customize it, just like customizing other
+ * Drupal templates such as page.html.twig or node.html.twig.
+ *
+ * Instead of overriding the theming for all fields, you can also just override
+ * theming for a subset of fields using
+ * @link themeable Theme hook suggestions. @endlink For example,
+ * here are some theme hook suggestions that can be used for a field_foo field
+ * on an article node type:
+ * - field--node--field-foo--article.html.twig
+ * - field--node--field-foo.html.twig
+ * - field--node--article.html.twig
+ * - field--field-foo.html.twig
+ * - field--text-with-summary.html.twig
+ * - field.html.twig
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - multiple: TRUE if a field can contain multiple items.
+ * - items: List of all the field items. Each item contains:
+ *   - attributes: List of HTML attributes for each item.
+ *   - content: The field item's content.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ * @see template_preprocess_field()
+ */
+#}
+{%
+  set title_classes = [
+    label_display == 'visually_hidden' ? 'visually-hidden',
+    'content-field__label',
+  ]
+%}
+
+{% if label_hidden %}
+  <ul class="list--no-bullet">
+    {% for item in items %}
+      <li{{ item.attributes.addClass('list-item') }}>{{ item.content }}</li>
+    {% endfor %}
+  </ul>
+{% else %}
+  <div{{ attributes.addClass('content-field--label-inline').setAttribute('role', 'presentation') }}>
+    <h2{{ title_attributes.addClass(title_classes) }}>{{ label }}</h2>
+    <div class="content-field__value">
+      <ul class="list--no-bullet">
+        {% for item in items %}
+          <li{{ item.attributes.addClass('list-item') }}>{{ item.content }}</li>
+        {% endfor %}
+      </ul>
+      <p class="help-viewing-documents">
+        <a href="/articles/help-viewing-documents">{{ 'Help viewing documents'|t }}</a>
```

Comment:
Would using the canonical node path for this be more robust than the aliased path? Haven't tried it but if Drupal is able to rewrite it at render time that'd be handy but if not it might look a bit bare and unhelpful.

Reply:
Changed it to this:
`<a href="{{ path('entity.node.canonical', {'node': '5643'}) }}">{{ 'Help viewing documents'|t }}</a>`